articles/liquid/liquid.md

```diff
@@ -0,0 +1,136 @@
+
+**Speed and Finality**: The network’s design emphasizes speed and reliability. Transactions on Liquid can achieve final settlement within two minutes, a significant improvement over Bitcoin. This is made possible through the streamlined block generation process managed by the functionaries, ensuring that transactions are processed swiftly and without the delays associated with mining on Bitcoin.
+
+**Asset Issuance Platform**: Beyond transacting in Bitcoin, Liquid provides functionalities for issuing a variety of digital assets. These assets can represent real-world entities like fiat currencies or securities, or they can be entirely digital creations such as tokens for specific ecosystems. The issuance process is integrated with Liquid’s privacy features, allowing issuers to maintain confidentiality over the operations while enabling the broad utility of these assets across the network.
+
+## **Trust & Security**
+The security framework of the Liquid Network, while bolstering its functionality and performance compared to the traditional Bitcoin blockchain, introduces unique trust dependencies and potential vulnerabilities. This section evaluates these elements critically, examining the implications of potential breakdowns and how they influence the overall integrity and reliability of the network.
+
+### **Trust Dependencies**
+
+1. **Federated Consensus Model**: Liquid’s federated consensus hinges on a pre-selected group of functionaries, which shifts trust from a decentralized model to a more centralized one. While this allows for faster transaction confirmations and improved privacy through Confidential Transactions, it also means that the network's security is heavily reliant on the integrity and reliability of these functionaries. If the functionaries were to act maliciously or if their systems were compromised, it could lead to manipulated transactions or loss of funds.
+2. **Functionaries as Block Signers and Watchmen**:
+    - **Block Signers**: The entire network trusts these functionaries to propose and sign blocks faithfully. If this trust is misplaced—through internal fraud or external hacking—transactions might be falsely validated or denied, impacting the network's reliability and users' assets.
+    - **Watchmen**: They manage the crucial two-way peg that allows BTC to be transferred in and out of the Liquid Network. A failure here, such as improper handling or theft of the pegged BTC, would directly threaten users' assets and could severely undermine trust in the platform's security.
+3. **Key Storage Modules**: The security of cryptographic keys is paramount as they underpin all transaction validations on the network. Any breach in these modules could potentially allow unauthorized access to sign transactions or alter the network operations.
```

Comment:
The key storage module is used by functionaries, and this is mentioned in `Security mechanism`. But here the context is missing so readers have no idea what key storage is, who use it and what it is used for.  

The technical detail part is originally designed for hard to understand and less important technical details. I think all of the content in technical details can be placed in the protocol quick scan as the overall content is  simple and easy to understand enough